utils/aur-check-update.py

```diff
@@ -55,21 +59,19 @@ def from_str(cls, s: str) -> Version:
         return cls(pkgver, pkgrel, epoch)
 
     @classmethod
-    async def from_file(cls, file: str) -> Version:
-        async with aiofiles.open(file) as fin:
+    async def from_file(cls, file: str | Path) -> Version:
```

Comment:
Is this standard syntax?
```suggestion
    async def from_file(cls, file: Union[str, Path]) -> Version:
```

Reply:
Please refer to this [PEP](https://www.python.org/dev/peps/pep-0585/#generics-erasure), and this synatx not working on py3.9 after `from __future__ import annotations` .